Makefile

```diff
@@ -199,8 +199,9 @@ eslint:
 	(cd templates;make eslint)
 
 jscoverage:
+	npm install --save-dev babel-jest @babel/core @babel/preset-env
```

Comment:
I'm not certain, but I don't think this line belongs here. IIRC npm install --save-dev is what you do the first time you want to introduce an npm package into a project. For both developer setup (docs/DeveloperSetup.rst) and our automated CI/CD, these packages will be installed via npm ci.

Please confirm and delete if you concur, otherwise please school me!

Reply:
Fixed, thanks for that!